lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -139,6 +148,33 @@ def get_request_id(headers, env)
           headers ||= {}
           headers['X-Request-Id'] || headers['X-Request-ID'] || env['HTTP_X_REQUEST_ID']
         end
+
+        private
+
+        REQUEST_SPAN_DEPRECATION_WARNING = %(
+          Use of :datadog_rack_request_span in the Rack env is DEPRECATED.
+          If you need the Rack request span, try using `Datadog.tracer.active_span`.
+          This key will be removed in version 0.13.0).freeze
```

Comment:
We can think about it. We can introduce this change after at least 2-3 releases so people know that something is going to break because internal symbols (that unfortunately are public-facing) are used.